format/Schema.fbs

```diff
@@ -178,6 +178,9 @@ table FixedSizeBinary {
 table Bool {
 }
 
+table RunEndEncoded {
+}
```

Comment:
Hmm, shouldn't this encode the run-end index size, like Int above or Decimal?

Reply:
I think, because the ends are a child array, and not a buffer, that this is not strictly needed.  RunEndEncoded ens up being more similar to Struct array in this regard.

Reply:
As it is currently implemented, the run-end index size isn't explicitly part of the type and is instead just an implementation detail in that the first child (containing the run-ends) can be any of int16/int32/int64. For example, note how List doesn't encode the child type and struct doesn't encode the child fields.

At least that's my understanding of why.